app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="ui middle aligned center aligned grid">
+	<div class="left aligned three wide column">
+
+		<h1 class="ui header">Sign-in</h1>
+		
+		<%= form_with scope: :session, url: login_path, class: 'ui large form', local: true do |f| %>
+			<div class="ui stacked segment">
+
+				<div class="field">
+					<%= f.label :email %>
+					<%= f.text_field :email %>
+				</div>
```

Comment:
* [ ] You should be using #form_for in `app/views/sessions` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/login_form_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication))